packages/dom/src/dom/is-input-or-text-area.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable jsdoc/valid-types */
+/**
+ * @param {Element} element
```

Comment:
Should this be `HTMLElement`?

Reply:
Nope, doesn't need to be that specific. This function will run on any `Element`.